csm_web/frontend/static/frontend/css/resource_aggregation.css

```diff
@@ -214,7 +218,7 @@
 	display: flex;
 	flex-direction: row;
 	justify-content: center;
-	margin: 20px auto;
+	margin: 20px 0px 20px 20px auto;
```

Comment:
Also here I don't think that `margin` accepts five fields; just omit the `auto` and It'll be good.